dissect/target/plugins/apps/shell/powershell.py

```diff
@@ -10,6 +10,7 @@
     [
         ("datetime", "mtime"),
         ("string", "command"),
+        ("varint", "order"),
```

Comment:
Would it make sense to move this above command for easier sorting in a terminal?